tests/pytests/test_geometry_sphere.py

```diff
@@ -262,17 +262,18 @@ def testFtInfo(env):
   ''' Test FT.INFO on Geoshape '''
   
   conn = getConnectionByEnv(env)
-  info_key_name = 'total_geoshapes_index_size_mb'
+  info_key_name = 'geoshapes_sz_mb'
   
   env.expect('FT.CREATE', 'idx1', 'SCHEMA', 'geom', 'GEOSHAPE', 'txt', 'TEXT').ok()
   env.expect('FT.CREATE', 'idx2_no_geom', 'SCHEMA', 'txt', 'TEXT').ok()
   res = to_dict(env.cmd('FT.INFO idx1'))
-  env.assertEqual(int(res[info_key_name]), 0)
+  cur_usage = float(res[info_key_name])
+  # env.assertEqual(int(res[info_key_name]), 0) # index is not lazily built. even an empty index consumes some memory
```

Comment:
Keep a comment (expecting the same initial size of an empty index) but remove commented out code?